roles/check_local_time/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- environment:
+    SC_HOST: "{{ sc_host }}"
+    SC_USERNAME: "{{ sc_username }}"
+    SC_PASSWORD: "{{ sc_password }}"
+    SC_TIMEOUT: "{{ sc_timeout }}"
```

Comment:
Those lines should be used only in integration tests. There, ansible-test will load variables from `integration-config.yml` - `sc_host` etc. Our modules are cofigured via environ `SC_HOST`, so first part of each integration test is that boilerplate code.

But in the actual roles and modules - do not do this. The `SC_HOST` environ must be set, or module needs to be invoked with `cluster_instance`.

Reply:
removed from role